client/src/stores/initApp.js

```diff
@@ -56,6 +57,17 @@ const initApp = async (state) => {
           update.selectedConnectionId = selectedConnectionId;
         }
       }
+
+      const qsConnectionName = queryString.parse(window.location.search)[
```

Comment:
Could we use the dot notation here for access? `queryString.parse(window.location.search).connectionName` should work.